### PR TITLE
python: Add support for fetching rights from a Daml 2.0 ledger.

### PR DESCRIPTION
**Pull request by da-tanabe**
python: Add support for fetching rights from a Daml 2.0 ledger.

This also includes minor fixes for running the test suite against a Daml 2.0 ledger.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 [tool.poetry]
 name = "dazl"
-version = "7.6.5"
+version = "7.7.0"
```
Comment (mschaef-da) [inline pyproject.toml:6]: Woot!

File: python/dazl/ledger/config/access.py [hunk 2]
```diff
@@ -412,17 +432,27 @@ def parties(p: Union[None, Party, Collection[Party]]) -> Collection[Party]:
...
+    """
+    Decode the claims section from a JSON Web Token (JWT).
+
+    Note that the signature is NOT verified; this is the responsibility of the caller!
+    """
     components = token.split(".", 3)
     if len(components) != 3:
         raise ValueError("not a JWT")
 
     pad_bytes = "=" * (-len(components[1]) % 4)
```
Comment (mschaef-da) [inline python/dazl/ledger/config/access.py:453]: If this was JavaScript, this line would have me _really_ worried...

File: python/dazl/ledger/grpc/channel.py [hunk 3]
```diff
@@ -74,3 +99,68 @@ def __call__(self, context: "AuthMetadataContext", callback: "AuthMetadataPlugin
...
+class GrpcAuthInterceptor(
+    UnaryUnaryClientInterceptor,
+    UnaryStreamClientInterceptor,
+    StreamUnaryClientInterceptor,
+    StreamStreamClientInterceptor,
+):
+    """
+    An interceptor that injects "Authorization" metadata into a request.
+
+    This works around the fact that the C++ gRPC libraries (which Python is built on) highly
```
Comment (mschaef-da) [inline python/dazl/ledger/grpc/channel.py:113]: These libraries seem _so_ helpful. :-)

**Review by mschaef-da — APPROVED** [review]

File: python/dazl/ledger/grpc/channel.py (hunk 3 above)
Comment (da-tanabe) [reply python/dazl/ledger/grpc/channel.py:113]: hahaha oh man… :-)

**Pull request merged**